includes/gateway/abstract-omise-payment-offline.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+defined( 'ABSPATH' ) || exit;
+
+require_once dirname( __FILE__ ) . '/class-omise-payment.php';
+
+/**
+ * @since 4.0
+ */
+abstract class Omise_Payment_Offline extends Omise_Payment {
+	/**
+	 * A string of Omise Source's type
+	 * (expecting for either 'paynow' or 'bill_payment_tesco_lotus').
```

Comment:
Comment should probably go as no doubt more offline payment methods will be added

Reply:
@jonrandy any suggestion? (can't think of other message than adding a word `at the moment`).
```php
/**
 * A string of Omise Source's type
 * 'bill_payment_tesco_lotus', 'paynow' (at the moment).
 */
```

Reply:
Just remove that line of the comment, the rest is fine